docker-compose.yml

```
@@ -1,4 +1,4 @@
-version: "3"
+version: '3'
```

Har sikkert ikke noe å si, føltes bare litt lite nødvendig
```suggestion
version: "3"
```